core/dbt/parser/models.py

```diff
@@ -86,11 +86,12 @@ def _flatten_attr(cls, node):
     def _safe_eval(self, node):
         try:
             return ast.literal_eval(node)
-        except (SyntaxError, ValueError, TypeError) as exc:
-            msg = validator_error_message(exc)
-            raise ParsingException(msg, node=self.dbt_node) from exc
-        except (MemoryError, RecursionError) as exc:
-            msg = validator_error_message(exc)
+        except (SyntaxError, ValueError, TypeError, MemoryError, RecursionError) as exc:
+            msg = validator_error_message(
+                f"Run into the following error when trying to literal_eval an arg \n{exc}\n"
```

Comment:
perhaps just "Error when trying to literal_eval an arg to ref() or source()" here?

Reply:
Updated!